docs/guides/completions.md

```diff
@@ -1,3 +1,42 @@
-Given a prompt, the model will return one or more predicted Completions. The Completions API can be run either
-synchronous or asynchronously (via Python `asyncio`); for each of these modes, you can also choose to stream token
-responses or not.
+# Overview
+
+**The Completions API** is used to generate text completions using the underlying language model. It supports both synchronous and asynchronous operations and offers an option to stream token responses.
+
+The API provides two key methods:
+
+1. **create()**: Synchronous method for creating a completion task.
+2. **acreate()**: Asynchronous method for creating a completion task.
+
+Both methods return either a **CompletionSyncV1Response** object or a stream of **CompletionStreamV1Response** objects, depending on the stream parameter.
+
+# Method Details
+**1. Create()**
+
+This method is used to create a completion task synchronously.
+
+Parameters:
```

Comment:
I don't think we want to duplicate the detailed API reference in here. Otherwise we run the risk of things getting out of date.

Maybe we want to treat this as more of a tutorial/cookbook that's based heavily on examples.

Reply:
https://platform.openai.com/docs/guides/gpt/completions-api

I might just add some examples, and add a "which model should I use?"